pyvista/plotting/axes_actor.py

```diff
@@ -402,6 +403,43 @@ def tip_type(self, tip_type: Union[TipType, int]):  # numpydoc ignore=GL08
         elif tip_type == AxesActor.TipType.SPHERE:
             self.SetTipTypeToSphere()
 
+    @property
+    def labels(self) -> Tuple[str, str, str]:  # numpydoc ignore=RT01
+        """Return or set the axes labels.
+
+        This property can be used as an alternative to :attr:`~x_axis_label`,
+        :attr:`~y_axis_label`, and :attr:`~z_axis_label`. The labels can be a single
+        string with exactly three characters (one for each of the x, y, and z axes,
+        respectively), or a sequence of three strings.
+
+        .. versionadded:: 0.44.0
+
+        Examples
+        --------
+        Set single-character axes labels.
+
+        >>> import pyvista as pv
+        >>> axes_actor = pv.AxesActor()
+        >>> axes_actor.labels = 'UVW'
+        >>> axes_actor.labels
+        ('U', 'V', 'W')
```

Comment:
The latter expression feels convenient, but this one feels confusing when viewed as a user. A short-sighted user like myself might specify “X AxisY AxisZ Axis” and so on. How about remove this and use only latter expression?

Reply:
Do you mean allow `Sequence[str]` but do not allow `str`? I do agree it is perhaps a bit too convenient. We can remove it.

Reply:
Yes. Let's remove it.